### ISSUE TITLE
Implement the Opportunities Tab in the JOBHUNT Fragment

### ISSUE THREAD

**Issue by riturajjain2000** [CSwala/CSwala-android#165]
The portals tab is already implemented, now the opportunities tab, containing information about job and Internship openings, is to be implemented

**Comment by GowthamGoush** [comment]
I'm interested in working on this, but from where do we fetch the data?

**Comment by riturajjain2000** [comment]
I think we should create a separate collection on the Firestore DB, any other ideas are welcome

**Comment by GowthamGoush** [comment]
So we have to look for a free API right?

**Comment by riturajjain2000** [comment]
yeah , see if you can fetch the data from google jobs or any other site like naukri.com or intershala

**Comment by GowthamGoush** [comment]
I couldn't find a free API for jobs based in India. So will web scraping with the help of a library be a good idea?

**Comment by riturajjain2000** [comment]
can you elaborate this process over discord channel, I mean what libraries will you use and how will you implement this workflow

**Comment by sitamadex11** [comment]
I want to work on this issue
